app/controllers/submit.js

```diff
@@ -792,6 +780,19 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             model.set('isPublished', true);
             node.set('public', true);
 
+            if (this.get('applyLicense')) {
+                let newCopyrightHolders = [''];
+                if (this.get('basicsLicense.copyrightHolders') && this.get('basicsLicense.copyrightHolders').length) {
+                    newCopyrightHolders = this.get('basicsLicense.copyrightHolders').split(',');
+                }
+                if (node.get('nodeLicense.year') !== this.get('basicsLicense.year') || node.get('nodeLicense.copyrightHolders') !== newCopyrightHolders) {
```

Comment:
This seems to be fine. If the copyright holders are changed, so is the license type. Then, either a different license that does not require copyright holders will be applied to the Node, so that would not be shown either way, or a license that does, in which case the form can't be submitted without putting somehting down for copyright Holders. There is no case where you are clearing node information that would be used.